src/lib/wallet/GoodWalletClass.js

```diff
@@ -369,10 +373,14 @@ export class GoodWallet {
 
   async getAmountAndQuantityClaimedToday(): Promise<any> {
     try {
-      const res = (await this.UBIContract.methods.getDailyStats().call()) || [ZERO, ZERO]
+      let stats = await this.UBIContract.methods.getDailyStats().call()
+      if (!stats) {
+        stats = [ZERO, ZERO]
+      }
+
```

Comment:
const [people, amount] = stats.map(value => value.toNumber())

return { people, amount }

Reply:
map doesn't work, because  stats it's not array

Reply:
what it actually returns ? probably it's some iterable ?
try `const [people, amount] = Array.from(stats).map(value => value.toNumber())` - probably this could help

Reply:
![image](https://user-images.githubusercontent.com/8187382/78998954-56156980-7b52-11ea-8b87-f2bf5e488de6.png)

Reply:
done